devel/export-dt-ini.lua

```diff
@@ -129,6 +129,7 @@ address('historical_figures_vector',globals,'world','history','figures')
 address('world_site_type',df.world_site,'type')
 address('active_sites_vector',df.world_data,'active_site')
 address('gview',globals,'gview')
+address('external_flag',globals,'game_extra','external_flag')
```

Comment:
this *may* change again in the future when/if the `gamest` padding gets into the base game.

Reply:
i'm going to be reluctant to rejoin the two halves of `game`; while I think Putnam has the best of intentions with her padding strategy, all it takes is for Toady to decide that another field is needed for some reason for the whole thing to get thrown out of whack again, and I judge that highly likely to happen at some time in the future.